### PR TITLE
[Word] (tutorial) Refresh

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Review by AlexJerabek — CHANGES_REQUESTED** [review]

**Comment by AlexJerabek** [comment]
(@ElizabethSamuel-MSFT, apologies that some of the comments are not on the lines they correspond to. GitHub wouldn't let me be more helpful.

File: docs/tutorials/word-tutorial.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ In this step of the tutorial, you'll programmatically test that your add-in supp
 
 1. Add the following function to the end of the file. Note:
 
-   - Your Word.js business logic will be added to the function that is passed to `Word.run`. This logic does not execute immediately. Instead, it is added to a queue of pending commands.
+   - Your Word.js business logic will be added to the function that is passed to `Word.run`. This logic does not execute immediately. Instead, it's added to a queue of pending commands.
 
    - The `context.sync` method sends all queued commands to Word for execution.
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/tutorials/word-tutorial.md:95]: Updated

File: docs/tutorials/word-tutorial.md [hunk 2]
```diff
@@ -74,28 +74,21 @@ In this step of the tutorial, you'll programmatically test that your add-in supp
...
     // Assign event handlers and other initialization logic.
-    document.getElementById("insert-paragraph").onclick = insertParagraph;
+    document.getElementById("insert-paragraph").onclick = () => tryCatch(insertParagraph);
     ```
 
-1. Add the following function to the end of the file. Note:
+1. Add the following functions to the end of the file. Note:
 
-   - Your Word.js business logic will be added to the function that is passed to `Word.run`. This logic does not execute immediately. Instead, it's added to a queue of pending commands.
+   - Your Word.js business logic will be added to the function that's passed to `Word.run`. This logic doesn't execute immediately. Instead, it's added to a queue of pending commands.
```
Comment (AlexJerabek) [inline docs/tutorials/word-tutorial.md:87]:
```suggestion
   - Your Word.js business logic will be added to the function passed to `Word.run`. This logic doesn't execute immediately. Instead, it's added to a queue of pending commands.
```

File: docs/tutorials/word-tutorial.md [hunk 3]
```diff
@@ -74,28 +74,21 @@ In this step of the tutorial, you'll programmatically test that your add-in supp
...
 
    - The `context.sync` method sends all queued commands to Word for execution.
 
-   - The `Word.run` is followed by a `catch` block. This is a best practice that you should always follow.
-
-   [!include[Information about the use of ES6 JavaScript](../includes/modern-js-note.md)]
+   - The `tryCatch` function will be used by all the functions interacting with the workbook from the task pane. Catching Office JavaScript errors in this fashion is a convenient way to generically handle any uncaught errors.
```
Comment (AlexJerabek) [inline docs/tutorials/word-tutorial.md:91]:
```suggestion
   - The `tryCatch` function will be used by all the functions interacting with the workbook from the task pane. Catching Office JavaScript errors in this fashion is a convenient way to generically handle uncaught errors.
```

**Review by AlexJerabek — APPROVED** [review]
Approving with a couple minor suggestions.

**Pull request merged**